docs/release_notes.rst

```diff
@@ -9,6 +9,12 @@ v2025.XX.x (2025-MM-DD)
 New Data
 ^^^^^^^^
 
+* Thanks to contributions from :user:`alexclippinger`, we've added cleaned EIA923
+  Schedule 8A Byproduct Disposition to the PUDL database as
+  :ref:`i_core_eia923__yearly_byproduct_disposition`. Once harvested, these tables will
+  eventually be removed from the database, but they are being published until then.
```

Comment:
Reworded to make it clearer that the data won't go away, just be in a different place.
```suggestion
  :ref:`i_core_eia923__yearly_byproduct_disposition`. Once harvested, this table will
  be replaced with a well-normalized version of the same data, but it is being published
  in this form until then.
```